packages/react/src/hooks/useAuth.ts

```diff
@@ -102,18 +122,36 @@ export const useAuth: UseAuth = () => {
       sessionId,
       userId,
       actor: null,
+      orgId: null,
+      orgRole: null,
       signOut,
       getToken,
     };
   }
 
   if (!!sessionId && !!userId) {
+    if (!!orgId && !!orgRole) {
+      return {
```

Comment:
How come we're not using ternaries for `orgId` and `orgRole` instead of the nested if?  Is this the only way to satisfy the discriminated union?

Reply:
I think that was it, but I'll double check

Reply:
Can't seem to make it work atm, but feel free to try it out

Reply:
If type inference with ternaries does not work in this case, lets refactor the conditions to make this more readable: 


```
  if (!!sessionId && !!userId && !orgId) { 
       ...
  }

  if (!!sessionId && !!userId && !!orgId) { 
       ...
  }
```

Reply:
Actually in the latter case we need to check for orgRole and slug as well for typescript to work, but done!